eks-anywhere-common/Addons/Partner/dynatrace/external-secret.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: external-secrets.io/v1beta1
```

Comment:
Fails with ExternalSecret/dynatrace/dynatrace-external-secret namespace not specified, error: namespaces "dynatrace" not found.
Please consider moving namespace create to the root level or moving the secret to the operator.

Reply:
I'll move it and update!

Reply:
moved in https://github.com/aws-samples/eks-anywhere-addons/pull/31/commits/06a4bb79e341bf68871997ebd3c013a7b87fb44b